includes/event/event-repository-interface.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Wporg\TranslationEvents\Event;
+
+use Exception;
+use Throwable;
+use WP_Error;
+
+class EventNotFound extends Exception {
+	public function __construct( Throwable $previous = null ) {
+		parent::__construct( 'Event not found', 0, $previous );
+	}
+}
+
+interface Event_Repository_Interface {
+	/**
```

Comment:
I suggest adding the explanation of what each function does (not only in this one), as the [wp_insert_post](https://github.com/WordPress/WordPress/blob/9ed35adffa36c706ed5e2642fd91e86c5d011f83/wp-includes/post.php#L4082) function the WordPress core does. Something simple as
```suggestion
	/**
	* Inserts or update an event.
```